src/views/HtmlDisplayView.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <HtmlDisplay :url="url" />
+</template>
+
+<script setup lang="ts">
+import HtmlDisplay from '@/components/general/HtmlDisplay.vue'
+import { useConfigStore } from '@/stores/config'
+import { computed } from 'vue'
+import { useRoute } from 'vue-router'
+
+const route = useRoute()
+
+const configStore = useConfigStore()
```

Comment:
This is almost but not quite duplicated, could this maybe be abstracted somehow?